### PR TITLE
fix: remove versions from provider block

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

```
Warning: Version constraints inside provider configuration blocks are deprecated
--
  |   |   |   | │
  |   |   |   | │   on .terraform/modules/jx/main.tf line 15, in provider "google":
  |   |   |   | │   15:   version = ">= 3.46.0, < 4.0.0"
  |   |   |   | │
  |   |   |   | │ Terraform 0.13 and earlier allowed provider version constraints inside the
  |   |   |   | │ provider configuration block, but that is now deprecated and will be
  |   |   |   | │ removed in a future version of Terraform. To silence this warning, move the
  |   |   |   | │ provider version constraint into the required_providers block.
  |   |   |   | │ (and 7 more similar warnings elsewhere)
```

### CODE REVIEW

File: modules/cluster/variables.tf [hunk 1]
```diff
@@ -80,7 +80,7 @@ variable "node_machine_type" {
 
 // https://cloud.google.com/compute/docs/machine-types
 variable "machine_types_cpu" {
-  type = map
+  type = map(any)
```
Comment (ankitm123) [inline modules/cluster/variables.tf:83]: This was added after running terraform fmt command.

**Review by tomhobson — APPROVED** [review]
LGTM

**Review by msvticket — APPROVED** [review]

**Pull request merged**